src/App.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+
+import TodoItemCreater from './TodoItemCreater';
+import TodoList from './TodoList';
+
+const App = () => {
+  const [todoItemList, setTodoItemList] = useState([]);
+
+  const handleSubmit = (todoItem) => {
+    setTodoItemList((prev) => [...prev, todoItem]);
```

Comment:
```suggestion
    setTodoItemList([...todoItemList, todoItem]);
```

callback으로 넘기지 않아도 바로 상태를 주입시키면 더 간단하게 표현이 가능할 것 같아요